tests/pipelines/test_pipelines_audio_classification.py

```diff
@@ -45,6 +47,9 @@ class AudioClassificationPipelineTests(unittest.TestCase):
     model_mapping = MODEL_FOR_AUDIO_CLASSIFICATION_MAPPING
     tf_model_mapping = TF_MODEL_FOR_AUDIO_CLASSIFICATION_MAPPING
 
+    if _run_pipeline_tests:
+        _dataset = datasets.load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
+
```

Comment:
do this in the class level instead of in `run_pipeline_test` method.

Therefore, inside each pytest process, this will be called only once.